src/TestFramework/DataFaker.php

```diff
@@ -208,6 +208,25 @@ public function subscriptionStatus()
     }
 
     /**
+     * Return a subscription billing state
+     *
+     * @return string
+     */
+    public function subscriptionBillingState()
+    {
+        $billingState = array(
+            'Unbilled',
+            'Good Standing',
+            'Free/Trial',
+            'In Retry',
+            'Unusable Payment Method',
+            'Billing Completed'
+        );
+        shuffle($billingState);
```

Comment:
instead of doing `shuffle`, can you use the $this->intBetween function to choose a random position? That way everything uses the number generator that was seeded in the constructor and you can just set the seed in one place to get consistent results every time